src/app/app.component.html

```diff
@@ -36,6 +55,7 @@
     </mat-nav-list>
 
     <app-popular-recipients
+        [ngStyle]="{ 'display': webmailSettings.showPopularRecipients ? 'block' : 'none' }"
```

Comment:
Why a style and not an ngIf? (just curious)

Reply:
ngIf is actually noticably slower when re-enabling it (takes a few hundred ms, seemingly), so I'd say the CSS is probably worth keeping for the sake of apparent "snapiness" of things. @gtandersen, what do you think?